packages/pluggable-widgets-tools/src/typings-generator/helpers.ts

```diff
@@ -17,3 +17,12 @@ function extractProps<P>(props: Properties, extractor: (props: Properties) => P[
 export function capitalizeFirstLetter(text: string): string {
     return text.charAt(0).toUpperCase() + text.slice(1);
 }
+
+export function commasAnd(arr: any[]) {
```

Comment:
`any` :(

Reply:
Does `string[]` make sense ?

Reply:
Anything except `any` 😁
`string[]` is fine :)